stratum/hal/lib/barefoot/bf_pipeline_utils.cc

```diff
@@ -28,6 +28,19 @@
 
 namespace stratum {
 namespace {
+// Helper function to see in an archive can be read
+::util::Status CheckArchive(const std::string& archive) {
```

Comment:
This should probably be named `IsArchive` and return `bool`.

Reply:
good point; fixed